pkgs/servers/nextcloud/default.nix

```diff
@@ -1,48 +1,63 @@
-{ lib, stdenvNoCC, fetchurl, nixosTests
-, nextcloud28Packages
-, nextcloud29Packages
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  nixosTests,
+  nextcloud28Packages,
+  nextcloud29Packages,
+  nextcloud30Packages,
 }:
```

Comment:
Please split this into another commit. Reviewing a mixture of behavioral and stylistic changes is always annoying.